OpenVBX/views/messages/details.php

```diff
@@ -124,6 +124,43 @@
 			<h3>Transcription</h3>
 			<?php endif; ?>
 			<div class="message-transcript"><?php echo (is_null($message->content_text) ? "(no transcription)" : $message->content_text) ?></div>
+			<?php if(substr($message->call_sid,0,2) == 'MM'):
+				echo "<br><br><h3>Media Attachements</h3>";
+				
+				$ci = & get_instance();
+				
+				require_once(APPPATH . 'libraries/twilio.php');
+				
+				$ci->twilio = new TwilioRestClient($ci->twilio_sid,$ci->twilio_token,$ci->twilio_endpoint);
+				
+				$media_url = "Accounts/{$this->twilio_sid}/Messages/{$message->call_sid}/Media.xml";
+				
+				$mediaObj = $ci->twilio->request($media_url, "GET");
+				
+				$media_xml = $mediaObj->ResponseXml;
+				
+				foreach($media_xml->MediaList->Media as $media):
```

Comment:
Nit pick: we can use the alternate syntax control when breaking from HTML, but once within PHP we should always use braces around control structures.